sdk/spring/azure-spring-cloud-context/README.md

```diff
@@ -1,9 +1,61 @@
 # Azure Spring cloud azure context client library for Java
+This package helps developers to finish the auto-configuration of Azure Context.
+
+[Source code][src] | [Package (Maven)][package] | [API reference documentation][refdocs]
 
-## Key concepts
 ## Getting started
+### Prerequisites
+- Java Development Kit (JDK) with version 8 or above
+- [Azure Subscription][azure_subscription]
+- [Maven][maven] 3.0 and above
+
+### Include the package
+[//]: # "{x-version-update-start;com.microsoft.azure:spring-cloud-azure-context;current}"
+```xml
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>spring-cloud-azure-context</artifactId>
+    <version>1.2.8-beta.1</version>
+</dependency>
+```
+[//]: # "{x-version-update-end}"
+
 ## Key concepts
+Azure contexts are PowerShell objects representing your active subscription to run commands against, and the authentication information needed to connect to an Azure cloud. With Azure contexts, Azure PowerShell doesn't need to reauthenticate your account each time you switch subscriptions.
```

Comment:
I don't think our azure-context is the PowerShell object 😄. But I believe they share the same abstraction.

Reply:
I updated this part, please help to check, thx